src/pages/HomePage/index.tsx

```diff
@@ -41,7 +42,7 @@ export default function HomePage() {
   });
   const navigate = useNavigate();
 
-  const isAllowedToSearch = searchText.length === 66;
+  const isAllowedToSearch = [66, 63].includes(searchText.length);
```

Comment:
Can you clarify why the change? It might be worth it to document here in the code too.

Reply:
The idea was to validate by the length of the address, but it makes more sense to use the real validation of bech32 in this case. I made a change and added a comment and changed the user feedback.